components/typography/test/typography.visual-diff.js

```diff
@@ -61,6 +61,16 @@ describe('d2l-typography', function() {
 			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
 		});
 
+		it('blockquote', async function() {
+			const rect = await visualDiff.getRect(page, '#blockquote', 0);
+			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+		});
+
+		it('blockquote-rtl', async function() {
+			const rect = await visualDiff.getRect(page, '#blockquote-rtl', 0);
+			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+		});
```

Comment:
So technically these visual diffs are only testing half the code -- the part in `typography.js`, but not the shared styles in `styles.js`. That's true for everything in this visual-diff file though, so I guess we can just follow along. But probably something for us (Gaudi) to resolve.

Reply:
Yeah was thinking the same thing. Might need a separate vdiff page for testing both scenarios.